spec/services/migration/migrators/participant_id_change_spec.rb

```diff
@@ -28,11 +28,23 @@ def setup_failure_state
         participant_id_change = ParticipantIdChange.first
         expect(participant_id_change).to have_attributes({
           user_id: User.find_by(ecf_id: ecf_resource1.user_id).id,
-          from_participant_id: User.find_by(ecf_id: ecf_resource1.from_participant_id).id,
-          to_participant_id: User.find_by(ecf_id: ecf_resource1.to_participant_id).id,
+          from_participant_id: ecf_resource1.from_participant_id,
+          to_participant_id: ecf_resource1.to_participant_id,
           created_at: ecf_resource1.created_at,
         })
       end
+
+      context "when corresponding from/to users do not exist" do
+        def create_npq_resource(ecf_resource)
+          user = create(:user, ecf_id: ecf_resource.user.id)
+
+          create(:participant_id_change, user:, from_participant_id: SecureRandom.uuid, to_participant_id: SecureRandom.uuid)
```

Comment:
Should we not be creating a participant ID change in ECF where the users don't exist in NPQ reg here?